fs/adaptive_fetch_image_layers.go

```diff
@@ -563,6 +564,7 @@ func newImageUnpackJob(imageDigest string, opts ...imageUnpackOption) *imageUnpa
 		concurrentDownloadsLimiter:       NewSemaphoreWithNil(unlimited),
 		concurrentUnpacksLimiter:         NewSemaphoreWithNil(unlimited),
 		layers:                           make(map[string][]*layerUnpackJob),
+		bufferPool:                       newbufferPool(64 * 1024),
```

Comment:
This is only 64KB so presumably this doesn't need to be configurable, but floating the idea of making it configurable anyway, just in case others disagree.